EIPS/eip-894.md

```diff
@@ -0,0 +1,39 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Definition
+The definition of bailouts in this EIP is state changes to the Ethereum blockchain, through a hardfork, that treat account holders in different ways.
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
```

Comment:
it's so good that ethereum is a centralized protocol defined and owned by a single organization

Reply:
The official Ethereum repositories are centralized, but the protocol is not.

Reply:
How are the official Ethereum repos centralized?

Reply:
If you look at this repo you will understand why:

https://github.com/popcorn-official/popcorn-app

Reply:
So DMCA takedowns are the problem?  This isn't a centralization problem with Ethereum, it's a problem with with Github.  Are you suggesting that we create a development platform on top of the protocol so that it too is decentralized?  I don't see how this is relevant to this EIP.

I think what @debris was getting at with their comment is that "standardized response" is something very few people are interested in, and the argument about official Ethereum repos being on Github has practically nothing to do with this.  It isn't comparable to popcorn-app IMO.

Reply:
I said that the protocol is decentralized and that's why people are wasting their time crying for bailouts.

Reply:
As a decentralized protocol run by miners, individuals are perfectly allowed to request that the protocol be changed for whatever reason. If the reason is understood and accepted by the miners, the change will happen. Period. End of discussion. That's literally all there is to it.

If you don't want these things to happen, you need to have a way to voice your objections.  In every other platform I can think of (centralized or otherwise), there are standards to how the conversation is conducted.  This is what the ERP EIP tried to create (and failed partially due to poor implementation), and what your EIP here seeks to undermine or eliminate.  It's literally counter-productive to your own goal.